src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
```

Comment:
Yeh it serves as documentation of how to use the python binding and end to end testing which for me makes more sense to have it in a different file.